src/Tasks/ContentReviewEmails.php

```diff
@@ -42,6 +45,16 @@ public function run($request)
         }
 
         ContentReviewCompatability::done($compatibility);
+
+        if (is_array($this->invalid_emails) && count($this->invalid_emails) > 0) {
+            $plural = count($this->invalid_emails) > 1 ? 's are' : ' is';
+            throw new Exception(
```

Comment:
```suggestion
            throw new RuntimeException(
```
We can be a little more specific here

Reply:
DONE